jartes/wikiblock
# Add block panel options

Fixes #5

## Description

We want to add some additional side panel options to the block, to control the iframe width, along with some Wikiloc built-in options, like showing the title, gallery, elevation, etc. 

## Changes in this PR
- Added the following block panel options (and also as block attributes): 
    - Map Width
    - Map Height
    - Map Type
    - Show title
    - Show measurements and elevation
    - Show location
    - Show images

## Test instructions

1. If you already have this plugin installed on your site, please deactivate it and remove it. 
2. Check out this branch `feature/panel-options`. 
3. Run the asset build: npm run build.
4. Run the zip build: npm run plugin-zip.
5. Upload the plugin zip to your test site and activate the plugin.
6. Edit or create a new post, and add a "Wikiblock" block. 
7. Add any Wikiloc trial ([example](https://es.wikiloc.com/rutas-senderismo/roca-de-sant-salvador-lelefant-miranda-de-santa-magdalena-100-cims-2-feec-ibp-82-hkg-03-04-2018-23747018)).
8. Check that on the right side that the mentioned options are being displayed. 
9. Check that after changing the options, the map is being updated on the front and on the editor.
10. Check the browser console for any block errors.

## Screenshots
![Markup on 2022-03-28 at 17:35:31](https://user-images.githubusercontent.com/1046943/160434600-161c30dd-b198-4bf1-90ae-d877b58f165c.png)
